tensorflow/lite/java/demo/app/src/main/AndroidManifest.xml

```diff
@@ -23,8 +23,6 @@
     <uses-feature android:name="android.hardware.camera" />
     <uses-feature android:name="android.hardware.camera.autofocus" />
 
-    <uses-sdk android:minSdkVersion="21" />
```

Comment:
We should keep this in the manifest, as this sample can also be built with bazel.

Reply:
Updated and squashed.  I added a [comment to the XML](https://github.com/tensorflow/tensorflow/pull/24255/files#diff-b2a1cafabbd44e7687f1ac02600d8da4R26
) to tell other people not to try and fix that warning. 

Thanks for the review!